sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -1078,6 +1229,16 @@ def __repr__(self):
             )[:1024]
         )
 
+    def to_dict(self):
+        return {
+            "model_id": self.model_id,
+            "status": self.status,
+            "training_started_on": self.training_started_on,
+            "training_completed_on": self.training_completed_on,
+            "model_name": self.model_name,
+            "properties": self.properties.to_dict()
```

Comment:
for v2.0 compatibility, we should check if it's `None` first

```suggestion
            "properties": self.properties.to_dict() if self.properties else None
```

Reply:
Agreed and done.